src/annotation/annotations_app/views/api.py

```diff
@@ -378,6 +378,59 @@ def image_annotations(image_id):
         )
 
 
+@app.route("/api/v0/annotations/", methods=['GET'])
+@app.route("/api/v0/annotations/<string:collection_id>", methods=['GET'])
+@app.route("/api/v0/annotations/<string:collection_id>/<string:image_id>", methods=['GET'])
```

Comment:
would it be more fun to pass `collection_id` and `image_id` as query parameters on `/annotations/`?

Reply:
I think this is a "filtered list" pattern  - show me all annotations (filter on this collection, and filter on that image_id)

Reply:
Thanks for the suggestion, I updated the API to reflect this.

@monkeypants  @koriaf: Just to make things complicated I also did some housekeeping on the annotations table, which required changing the initial alembic schema- you will need to rebuild your db.